src/ui/server/constants/integrity.ts

```diff
@@ -1,7 +1,7 @@
 export const INTEGRITY = {
   JS: 'sha512-BFOXngFJ+9msC1y5Q1LD6AG+LG635vp8Lfv/+HVCUU8w/N8VWH13wdTyZeLGvu4d7zoTz83nfz329ENYfe9x5Q==',
```

Comment:
Is this `main.js`, if yes does SRI for this also needs to be updated give `main.js` has also been updated?

Reply:
I don't believe it does @abhi-markan , `main.js` purely imports SCSS, there's no actual client side JS used in this file.

I think we should remove this integrity hash instead